dotnet/src/Agents/OpenAI/Internal/AssistantThreadActions.cs

```diff
@@ -353,7 +353,7 @@ public static async IAsyncEnumerable<StreamingChatMessageContent> InvokeStreamin
         string threadId,
         IList<ChatMessageContent>? messages,
         RunCreationOptions? invocationOptions,
-        string? behaviorsContext,
+        string? providersContext,
```

Comment:
I normally see `context` associating to an object not a `string`. Might suggest another atribute name here maybe?